### PR TITLE
Feat/scenario f

### PR DESCRIPTION
**Pull request by ioanaroceanu2000**

### CODE REVIEW

File: integration/test/scenarios/ScenarioF.sol [hunk 1]
```diff
@@ -0,0 +1,756 @@
...
+
+    uint128 public productId;
+    uint128 public marketIdAave;
+    uint128 public marketIdGlp;
+    uint32 public maturityTimestampAave;
+    uint32 public maturityTimestampGlp;
+    int24 public initTickAave;
+    int24 public initTickGlp;
+
+    struct Something {
```
Comment (0xZenus) [inline integration/test/scenarios/ScenarioF.sol:43]: what do we need this for?

File: integration/test/scenarios/ScenarioF.sol [hunk 2]
```diff
@@ -0,0 +1,756 @@
...
+            accountIds
+        );
+    }
+
+    function setUp() public {
+        super.datedIrsSetup();
+        marketIdAave = 1;
+        marketIdGlp = 2;
+        maturityTimestampAave = uint32(block.timestamp) + 365 * 86400; // in 1 year
+        maturityTimestampGlp = uint32(block.timestamp) + 365 * 86400 / 2; // in 1 year
```
Comment (0xZenus) [inline integration/test/scenarios/ScenarioF.sol:89]: in 1/2 years

Comment (ioanaroceanu2000) [reply integration/test/scenarios/ScenarioF.sol]: fixed in [0bd6c16](https://github.com/Voltz-Protocol/v2-core/pull/184/commits/0bd6c16576b1ff500d29d6db262ea8b160c74096)

File: integration/test/scenarios/ScenarioF.sol (hunk 1 above)
Comment (ioanaroceanu2000) [reply integration/test/scenarios/ScenarioF.sol]: ahh forgot to delete it. removed in [0bd6c16](https://github.com/Voltz-Protocol/v2-core/pull/184/commits/0bd6c16576b1ff500d29d6db262ea8b160c74096)

**Review by 0xZenus — APPROVED** [review]

**Pull request merged**